contracts/PetPark.sol

```diff
@@ -4,4 +4,93 @@ pragma solidity ^0.8.0;
 
 contract PetPark {
 
+    address  contractOwner;
+    enum AnimalType {
+        NONE, FISH, CAT, DOG, RABBIT, PARROT
+    }
+    enum Gender{
+        MALE, FEMALE
+    }
+    struct Animal {
+        AnimalType animalType;
+        uint count;
+    }
+
+    struct Borrower {
+        Gender gender;
+        uint age;
+        uint petCount;
+        bool exists;
```

Comment:
Since storage in solidity is expensive, instead of using `exists` can't you use `age > 0` to make this check?